packages/smart-wallet/src/smartWallet.js

```diff
@@ -277,6 +286,9 @@ const SmartWalletKit = defineVirtualFarClassKit(
             balance: a,
           })),
           offerToUsedInvitation: mapToRecord(offerToUsedInvitation),
+          offerToPublicSubscriberPaths: mapToRecord(
+            offerToPublicSubscriberPaths,
+          ),
```

Comment:
hm... I mostly see maps shared in `.entries()` form. Too bad I didn't think of that for `offerToUsedInvitation`.